src/components/common/ListPeopleAndRoles.vue

```diff
@@ -2,62 +2,97 @@
   <div id="list-people-roles">
     <section :class="{ 'invalid-section': showErrorSummary }">
       <!-- Summary Warning -->
-      <div v-if="isSummary && showErrorSummary" class="people-roles-invalid-message">
+      <div
+        v-if="isSummary && showErrorSummary"
+        class="people-roles-invalid-message"
+      >
         <span>
           <v-icon color="error">mdi-information-outline</v-icon>
           <span class="error-text mx-1">This step is unfinished.</span>
-          <router-link v-if="isIncorporationFiling"
+          <router-link
+            v-if="isIncorporationFiling"
             id="router-link"
             :to="{ path: `/${RouteNames.INCORPORATION_PEOPLE_ROLES}` }"
           >Return to this step to finish it</router-link>
-          <router-link v-if="isRegistrationFiling"
+          <router-link
+            v-if="isRegistrationFiling"
             id="router-link"
             :to="{ path: `/${RouteNames.REGISTRATION_PEOPLE_ROLES}` }"
           >Return to this step to finish it</router-link>
-          <router-link v-if="isFullRestorationFiling || isLimitedRestorationFiling"
+          <router-link
+            v-if="isFullRestorationFiling || isLimitedRestorationFiling"
             id="router-link"
             :to="{ path: `/${RouteNames.RESTORATION_APPLICANT_INFORMATION}` }"
           >Return to this step to finish it</router-link>
         </span>
       </div>
 
       <!-- List Display Section -->
-      <div id="people-roles-list" v-if="personList.length > 0">
+      <div
+        v-if="personList.length > 0"
+        id="people-roles-list"
+      >
         <!-- List Headers -->
-        <v-row class="people-roles-header" no-gutters>
-          <v-col v-for="(title, index) in tableHeaders" :key="index">
+        <v-row
+          class="people-roles-header"
+          no-gutters
+        >
+          <v-col
+            v-for="(title, index) in tableHeaders"
+            :key="index"
+          >
             <span>{{ title }}</span>
           </v-col>
 
           <!-- conditional empty alignment column -->
           <v-col v-if="showAlignmentColumn" />
 
           <!-- conditional action columns (no header) -->
-          <v-col v-if="!isSummary" class="actions-width" />
+          <v-col
+            v-if="!isSummary"
+            class="actions-width"
+          />
         </v-row>
 
         <!-- List Content -->
         <v-row
-          class="people-roles-content"
           v-for="(orgPerson, index) in personList"
           :key="index"
+          class="people-roles-content"
           no-gutters
         >
           <v-col class="name-column d-flex text-break">
             <div class="pr-2">
-              <v-icon color="gray9" v-if="isPerson(orgPerson)">mdi-account</v-icon>
-              <v-icon color="gray9" v-if="isOrg(orgPerson)">mdi-domain</v-icon>
+              <v-icon
+                v-if="isPerson(orgPerson)"
+                color="gray9"
```

Comment:
FYI, this is a Vuetify theme colour that we declared in main.ts. This is not a CSS variable as used elsewhere.

Reply:
So Sev from what I discovered, If we're using colors in the template, we use the ones declared in main.ts. If we're using colors in the the style block, we use the ones declared in the styles file. Right?

Reply:
Close! If we're using colours in a Vuetify component then we use the ones in main.ts. If we use a class to set colours then we use the CSS variables.

Reply:
OK, noted. Thanks!

Reply:
Hongjng, you can resolve this comment after you've read it :)